packages/wrangler/src/__tests__/pages/deploy.test.ts

```diff
@@ -34,6 +37,7 @@ describe("pages deploy", () => {
 	beforeEach(() => {
 		actualProcessEnvCI = process.env.CI;
 		process.env.CI = "true";
+		setIsTTY(false);
```

Comment:
This is more reliable than the `process.env.CI` setting (which wasn't working properly locally). I think we can remove the `process.env.CI` setting in a followup (I didn't remove it in this PR in order to minimise the diff)